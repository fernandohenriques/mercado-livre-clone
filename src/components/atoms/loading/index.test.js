import React from 'react';
import { mount } from 'enzyme';
import { render } from '@testing-library/react';
import Loading from './index';

describe('<Loading />', () => {
  it('Snapshot testing', () => {
    const { asFragment } = render(<Loading />);

    expect(asFragment()).toMatchSnapshot();
  });

  it('Should use props className as CSS class', () => {
    const wrapper = mount(<Loading />);
    const div = wrapper.find('div');

    expect(div).toHaveLength(1);
    expect(div.hasClass('loading')).toBe(true);
  });
});
