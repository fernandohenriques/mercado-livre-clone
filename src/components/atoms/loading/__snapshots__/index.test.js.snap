// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> Snapshot testing 1`] = `
<DocumentFragment>
  <div
    class="loading"
  />
</DocumentFragment>
`;
